src/test/java/CowTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
```

Comment:
this is for Cow.java

Reply:
I just forgot to change it... but I'll count it as an intended error. :))